.eslintrc.json

```diff
@@ -16,6 +16,7 @@
         "no-console": "off",
         "func-names": "off",
         "no-alert": "off",
-        "prefer-const": "off"
+        "prefer-const": ["error", {"destructuring": "all"}],
```

Comment:
I have added  that to warn us if `let` is used and we never resigned that variable.... `{"destructuring": "all"}` because  of the following....
```javascript
/*eslint prefer-const: ["error", {"destructuring": "all"}]*/
/*eslint-env es6*/

// 'b' is never reassigned, but all of `a` and `b` should not be const, so those are ignored.
let {a, b} = obj;
a = a + 1;
```